.storybook/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+module.exports = ({ config }) => {
```

Comment:
Does storybook use this file by default? I always thought it only used its own config internally and through `webpackFinal()`
I would have expected to see
```js
// .storybook/main.js
module.exports = {
  webpackFinal: require('./webpack.config.js')
  ...
}
```
or something

Reply:
I couldn't find where this is magically loaded either. I based this TS setup off of https://github.com/gnosis/safe-react-components/tree/development/.storybook
Nowhere I see this file mentioned directly.